code/modules/RCD/schematics/pipe.dm

```diff
@@ -382,8 +382,8 @@
 /datum/rcd_schematic/pipe/disposal/attack(var/atom/A, var/mob/user)
 	to_chat(user, "Building Pipes ...")
 	playsound(get_turf(user), 'sound/machines/click.ogg', 50, 1)
-	if(!do_after(user, A, 20))
-		return 1
+//	if(!do_after(user, A, 1))
+//		return 1
```

Comment:
@Skullyton this is the modular RCD framework. In the schematic activation procs, `1` means nondescriptive error, `string` means descriptive error, and `0/null` means everything was fine.